app/api/routes/profanity.py

```diff
@@ -17,6 +22,7 @@ def profanity_check_fasttext(payload: ProfanityCheckRequest):
     result = check_profanity_fasttext(payload.text)
     return result  # Return dict directly
 
+
 @router.post(
     "/llm",
```

Comment:
This can be named as profanity_validator?

Reply:
Ok, I'll rename it.